src/constant.js

```diff
@@ -0,0 +1,23 @@
+export const MAX_CAR_COUNT = 50;
+export const MAX_TRY_COUNT = 500;
+
+export const CONSOLE_MESSAGE = Object.freeze({
+  CAR_INPUT_MESSAGE:
+    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
+  TRY_COUNT_INPUT_MESSAGE: '시도할 횟수는 몇 회인가요?\n',
+  GAME_START_MESSAGE: '\n실행 결과',
+  ANNOUNCE_WINNER_MESSAGE: '최종 우승자 : ',
+});
+
+export const ERROR_MESSAGE = Object.freeze({
```

Comment:
```suggestion
export const MESSAGES = Object.freeze({
```
재영님께서 만들어주신 유틸 함수 `errorString`을 사용해야 앞에 `[ERROR]` 접두사가 붙어서 에러 메세지가 된다고 생각해서 여기는 `MESSAGES`라고만 정의해서 조금 더 범용성있게 사용하는건 어떨까요??

예를 들면 실제 웹사이트 인풋에 입력할 때 인풋 창 아래에 안내 메세지로 `경주할 자동차는 2대 이상이어야 합니다. 쉼표(,)로 구분해서 2대 이상 입력해주세요.`로 범용적인 메세지로 사용할 수 있을 것 같아요

Reply:
오 그렇군요. 좋은 의견 감사합니다